Dockerfile

```
@@ -0,0 +1,20 @@
+FROM golang:1.19 AS builder
+
+WORKDIR /code
+
+# this is broken out from call to `make` to improve docker caching
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
+
+# run main build
+COPY . .
+RUN  make dev
+
+FROM builder
```

I forgot that to really get the size savings the final stage needs to start from some other base image.
```suggestion
FROM alpine:latest AS prod
```
I am pretty sure that will still work...